src/Symfony/Component/Security/Http/HttpUtils.php

```diff
@@ -150,7 +150,16 @@ public function generateUri($request, $path)
         // fortunately, they all are, so we have to remove entire query string
```

Comment:
Should we update this comment too?

Reply:
dedicated comment added just before fragment handling